rts/motoko-rts-tests/src/gc/random.rs

```diff
@@ -52,9 +52,21 @@ pub(super) fn generate(seed: u64, max_objects: u32) -> TestHeap {
         })
         .collect();
 
+    // Same as roots
+    let region0_ptr_loc: Vec<ObjectIdx> = (0..n_objects)
```

Comment:
Right, it would actually be two `ObjectIdx`, one for the region object and one for its internal `vec_pages` (a blob).
This is different to the continuation table. As for the root collection in the GCs (in main RTS implementation), the handling of `region0` is analogous to the continuation table, as the GC analyzes the structure behind the roots.